### PR TITLE
overview, keystats, news testing added

### PR DESCRIPTION
**Pull request by slcfv3**
All versions merged together, and testing framework is added for overview, keystats, news.

### CODE REVIEW

File: stock-tracker/src/__test__/keyStats.test.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    it('should render correct value for volume', () => {
+        component(store)
+        const volume = screen.getByTestId('volume')
+    expect(volume.innerHTML).toBe(" 1138432 ");
+    });
+
+    it('should render correct value for market cap', () => {
+        component(store)
+        const marketCap = screen.getByTestId('market-cap')
+    expect(marketCap.innerHTML).toBe(" 30531041613 ");
```
Comment (mhamdar) [inline stock-tracker/src/__test__/keyStats.test.js:118]: I think it would be better to not pass in values to toBe() explicitly, but rather get them from the state in our store (e.g. toBe(initialState.keyState.marketCap)), so we could always change the state to test different edge cases without breaking our tests.
